recipe/meta.yaml

```diff
@@ -13,7 +13,10 @@ source:
 
 build:
     number: 1
-    skip: True  # [py35 and win32]
+    features:
+     - vc9  # [win and py27]
+     - vc10  # [win and py34]
+     - vc14  # [win and py35]
```

Comment:
Not sure if these are necessary. I never specified features before, but I guess we should start doing that.